docs/developer/dependencies.rst

```diff
@@ -28,7 +28,6 @@ Run
 
   - ``appdirs`` (MIT) (system application directory helper, used in user configuration loading)
   - ``ipywidgets`` * (BSD-3-Clause) (interactive Jupyter notebook widgets)
-  - ``mantid-framework`` * (GPL-3.0-or-later) (used for Mantid interoperability)
   - ``matplotlib`` * (custom permissive) (plotting library)
   - ``numpy`` >=1.15.3 (custom permissive)
   - ``Pillow`` (HPND) (Python Imaging Library, used in plot_3d to create colorbar)
```

Comment:
`Pillow` is no longer used. See #1771 
I think it would be super if we could somehow centralise all our dependencies in one place, between docs, conda env files, conda build metadata, etc...
I don't know if we could automatically generate these files, or just have a single file...

Reply:
We could add the licence and description as an inline comment in the yaml file and then link to, or print the yaml file, in the docs. Alternatively we could at least add a manual check that they are all in sync to the deployment procedure.

Reply:
:+1:  Removed Pillow and psutil after grepping to be sure they aren't used anywhere.